Source/Core/Core/ConfigManager.h

```diff
@@ -297,6 +297,7 @@ struct SConfig : NonCopyable
   std::string m_strMovieAuthor;
   unsigned int m_FrameSkip;
   bool m_DumpFrames;
+  int m_DumpFramesBitrate;
```

Comment:
This should likely be an `s64` since AVCodecContext's `bit_rate` member is an `int64_t`.